cpp/open3d/pipelines/integration/UniformTSDFVolume.cpp

```diff
@@ -294,6 +294,65 @@ std::shared_ptr<geometry::VoxelGrid> UniformTSDFVolume::ExtractVoxelGrid()
     return voxel_grid;
 }
 
+std::vector<Eigen::Vector2d> UniformTSDFVolume::ExtractVolumeTSDF()
+        const {
+    std::vector<Eigen::Vector2d> sharedvoxels_;
+    sharedvoxels_.resize(voxel_num_);
+
+    for (int x = 0; x < resolution_; x++) {
```

Comment:
can we use 
```
#ifdef _WIN32
#pragma omp parallel for schedule(static)
#else
#pragma omp parallel for collapse(3) schedule(static)
#endif
```
here to make this computation parallel on CPU?

Reply:
Hi, These changes might significantly speed up the performance. 
For your reference: `collapse(3)`, makes the nested for loops in this structure collapse into one and parallel, however the support might be missing in OpenMP2.0, because of which we avoid it using for Windows build.

Thanks.